src/telemetry/telemetry.go

```diff
@@ -26,7 +26,7 @@ var (
 
 func main() {
 	flag.Var(userAuth, "client_auth", "Client auth mode(s) - cert,password,jwt")
-	flag.BoolVar(&gnmi.RbacDisable, "rbac-disable", false, "Enabled/Disable RBAC")
+	flag.BoolVar(&gnmi.RbacDisable, "rbac_disable", false, "Disable RBAC (Default Enabled)")
```

Comment:
Doh, same comment as in my other review about the ambiguity in the "help text" here.